### ISSUE TITLE
Possible improvements

### ISSUE THREAD

**Issue by mctb32** [codecat/asdbg#2]
Some ideas for improvements to the library and debugger gui:
* focus / flash app when breakpoint is hit
* add Visual Studio style keyboard shortcuts to basic flow commands (Resume F5, Step Over F10, etc.). 
* clicking on items in Call Stack should jump to this file/position if possible
* view global variables
* make the pane with the scripts treeview resizable 9or slightly larger)
* Add “Save” option to the script editor, to actually modify the scripts. This could send a message to the host app, so that it could actually reload and rebuild the scripts via a callback. 
* Allow working with multiple AS contexts

**Comment by codecat** [comment]
All very good points, thanks!

Didn't I already implement the call stack click to jump to location? Perhaps it's double click. I know I needed that at some point too so I added it.

**Edit**: Ah, I remember now, I didn't implement that because it would also require showing local variables for the other stack frames. Not very hard to add to the host code though.

**Comment by mctb32** [comment]
Yeah, it seems jump to location is not there yet. We have #include support in our scripts, so the same script can have sections from various files in the same callstack. It looks good (the filenames displayed in the callstack are allright) so we're just missing the jump to location function.

**Comment by codecat** [comment]
I think if you're using the `CScriptBuilder` addon it'll handle the filenames and line numbers just fine when using `#include`, indeed.
